### DISCUSSION TITLE
Map fixed and moving images to their half-way space

### DISCUSSION THREAD

**Discussion by devDonnn** [ANTsX/ANTs#1313] [Q&A] [answered]
I use the following command to register two T1w images:

`antsRegistrationSyNQuick.sh 
-d 3 
-f vol1.nii.gz 
-m vol2.nii.gz 
-o output_ 
-t so 
-n 4 
-j 1 `

Now, for my particular application I want to apply warps for both images - fixed and moving - so these end up being co-registered as well. I see that I get both images warped but either image1 is mapped to image2 space or image2 mapped to image1 space. Instead, I would like to apply (half?) warps on both so both undergo warps (AND interpolation) and are well-registered too (same space). Is this possible? This is analogous to FreeSurfer's mri_robust_register tool where both images are co-registered to their half-way space.

**Comment by ntustison** [comment, marked as answer]
``unbiased_pairwise_registration.sh``
